src/handlers/zodSchemas.ts

```diff
@@ -0,0 +1,166 @@
+import { z } from 'zod';
+import { Type as _Type } from '../../appsync';
+const Type = { ..._Type };
```

Comment:
This weird hack is necessary so jest can recognise the `Type` enum, otherwise it throws an error saying it's undefined :(